logging/cyclicWriter.go

```diff
@@ -51,6 +64,45 @@ func MakeCyclicFileWriter(liveLogFilePath string, archiveFilePath string, sizeLi
 	return &cyclic
 }
 
+func (cyclic *CyclicFileWriter) getArchiveFilename(now time.Time) string {
+	buf := strings.Builder{}
+	cyclic.archiveFilename.Execute(&buf, struct {
+		Year      string
+		Month     string
+		Day       string
+		Hour      string
+		Minute    string
+		Second    string
+		EndYear   string
+		EndMonth  string
+		EndDay    string
+		EndHour   string
+		EndMinute string
+		EndSecond string
+	}{
+		fmt.Sprintf("%04d", cyclic.logStart.Year()),
```

Comment:
So this is misleading as it's the start time for the current session, not when the log file was created.  Do you see a use-case for the logStart value?

Reply:
it probably won't be more than a couple seconds in advance of the first log line and it will accurately be the start of subsequent log chunks.

Reply:
Sure but its value as the start of the last log chunk seems dubious.  Don't really care since i would just use the NOW time and assume it starts at the previous archive's NOW.